packages/viewer/src/lib/mappers/streamAttributes.ts

```diff
@@ -79,20 +79,30 @@ const toPbFeatureHighlight: M.Func<
   })
 );
 
+const toPbNoDefaultLights: M.Func<boolean | undefined, boolean> =
+  M.defineMapper(
+    (noDefaultLights) => {
+      return noDefaultLights ? noDefaultLights : false;
+    },
```

Comment:
Minor nit: a small shortcut here (assuming `noDefaultLights` initially is `boolean | undefined`) is to use `!!noDefaultLights`, which if the value is `undefined` will coerce it to false for the boolean operation. Then the mapper could just be `(noDefaultLights) => !!noDefaultLights`